### PR TITLE
Fix bugs and support Gitlab API v4

### PR DESCRIPTION
**Pull request by elwerene**

### CODE REVIEW

**Comment by elwerene** [comment]
@fnogatz can you please have a look? :)

**Comment by fnogatz** [comment]
Hi @elwerene, thank you for the PR and the reminder. It's been a long time, I worked on this project. Just let me have a look at the old code the next days... :D

File: index.js [hunk 1]
```diff
@@ -124,6 +124,7 @@ function generateIssue (body, url, options) {
   var now = new Date()
 
   var description = [
+    options.mention ? options.mention : '',
```
Comment (fnogatz) [inline index.js]: In the long run it might be better to just allow to configure the whole text at all :thinking:

**Comment by fnogatz** [comment]
I did the following updates:
- Upgrade `gitlab` dependency to version 4.2
- Add an example webpage in `/example`
- Use JavaScript Standard Style

Anything missing from your commits? :)

Thank you again for proposing the changes!

**Comment by elwerene** [comment]
Update body-parser to version 1.18.3.

**Comment by fnogatz** [comment]
Done.

**Pull request closed without merging**